src/components/CompareResults/ModeInterpretation.tsx

```diff
@@ -0,0 +1,72 @@
+import { Box } from '@mui/material';
+
+import { MANN_WHITNEY_U } from '../../common/constants';
+import { useAppSelector } from '../../hooks/app';
+import { Colors } from '../../styles/Colors';
+import { MannWhitneyResultsItem } from '../../types/state';
+import { TestVersion } from '../../types/types';
+
+export const ModeInterpretation = ({
+  result,
+  testVersion,
+}: {
+  result: MannWhitneyResultsItem;
+  testVersion: TestVersion;
+}) => {
+  if (!result || !result.silverman_kde || testVersion !== MANN_WHITNEY_U) {
+    return null;
+  }
+
+  function getStyles(theme: string) {
+    const backgroundColor =
+      theme === 'light' ? Colors.Background300 : Colors.Background300Dark;
+
+    return {
+      backgroundColor,
+      display: 'block',
+      alignItems: 'center',
+      margin: '15px',
+      borderRadius: '5px',
+      padding: 2,
+    };
+  }
+  const mode = useAppSelector((state) => state.theme.mode);
+
+  // const result?.silverman_kde?.mode_comments
+  return (
+    <Box sx={getStyles(mode)}>
+      <table>
+        <thead>
+          <tr
+            style={{
+              display: 'grid',
+              gridTemplateColumns: '2.75fr 1fr 1.25fr',
+              gap: 4,
+              textAlign: 'left',
+            }}
+          >
+            <th></th>
```

Comment:
We should go ahead and add the `start` and `end` columns for the`mode start` and `mode end` values. This will improve the table format and UI.

Reply:
Sounds good, I'll do that